src/picturefill.js

```diff
@@ -1272,13 +1272,13 @@
 		return match;
 	};
 
-	ri.parseSets = function( element, parent ) {
+	ri.parseSets = function( element, parent, options ) {
```

Comment:
Yeah, undefined options argument is handeled here:
https://github.com/scottjehl/picturefill/blob/c1e95596dfc83b493043feb910dcbfb89a6a689f/src/picturefill.js#L175